llvm/include/llvm/CodeGen/GlobalISel/GenericMachineInstrs.h

```diff
@@ -36,13 +36,13 @@ class GenericMachineInstr : public MachineInstr {
   }
 
   bool hasPoisonGeneratingFlags() const {
-    return getFlags() & (NoUWrap | NoSWrap | IsExact | Disjoint | NonNeg |
-                         FmNoNans | FmNoInfs);
+    return getFlags() & (NoUWrap | NoSWrap | NoUSWrap | IsExact | Disjoint |
+                         NonNeg | FmNoNans | FmNoInfs);
```

Comment:
We probably should define a constant mask for these somewhere instead of maintaining these 2 lists